Makefile

```diff
@@ -251,6 +251,11 @@ test.clean:
 # travis
 # ------
 
+PHONY += travis-pyenv
+travis.test:
```

Comment:
`travis.codecov` is misleading: we don't use travis anymore, but [github actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions).

A generic name can be `ci.test` (so why don't have to rename the target if we move away from github actions).

Reply:
Good point .. renamed to `ci.test` (https://github.com/searx/searx/pull/2524/commits/b53b91e1c4c7034f7777426f1679f6b68d79a487).